packages/flutter_driver/lib/src/driver.dart

```diff
@@ -211,15 +211,21 @@ class FlutterDriver {
     return await _sendCommand(new Scroll(finder, dx, dy, duration, frequency)).then((Map<String, dynamic> _) => null);
   }
 
+  /// Tell the driver to ensure that the widget located by [finder] has been
+  /// scrolled completely into view. See [Scrollable.ensureVisible].
+  Future<Null> scrollToVisible(SerializableFinder finder) async {
+    return await _sendCommand(new ScrollToVisible(finder)).then((Map<String, dynamic> _) => null);
+  }
+
   /// Returns the text in the `Text` widget located by [finder].
   Future<String> getText(SerializableFinder finder) async {
     return GetTextResult.fromJson(await _sendCommand(new GetText(finder))).text;
   }
 
   /// Starts recording performance traces.
-  Future<Null> startTracing() async {
+  Future<Null> startTracing({ String categories: '[all]' }) async {
```

Comment:
Also, let's add a unit-test to make sure we're passing this parameter correctly (see [flutter_driver_test.dart](https://github.com/flutter/flutter/blob/master/packages/flutter_driver/test/flutter_driver_test.dart#L200))